cli/server.go

```diff
@@ -814,6 +815,15 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 					return xerrors.Errorf("register agents prometheus metric: %w", err)
 				}
 				defer closeAgentsFunc()
+
+				if coderAPI.LicenseMetrics == nil {
+					return xerrors.Errorf("developer error: LicenseMetrics should be not nil")
+				}
+				closeLicenseMetricsFunc, err := coderAPI.LicenseMetrics.Collect(ctx)
```

Comment:
I don't think we should be registering license metrics in AGPL Coder.  The Entitlement pointer will never be set, all the gauges will be 0, and you'll get a periodic logger warnings about "entitlements have not been loaded yet"

Reply:
I moved the MetricsCollector to `enterprise`.